internal/operands/node-labeller/resources.go

```diff
@@ -0,0 +1,275 @@
+package node_labeller
+
+import (
+	"kubevirt.io/ssp-operator/internal/common"
+
+	secv1 "github.com/openshift/api/security/v1"
+	apps "k8s.io/api/apps/v1"
+	core "k8s.io/api/core/v1"
+	rbac "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Resources from node-labeller
+const (
+	ServiceAccountName       = "kubevirt-node-labeller"
+	DaemonSetName            = "kubevirt-node-labeller"
+	ConfigMapName            = "kubevirt-cpu-plugin-configmap"
+	ClusterRoleName          = "kubevirt-node-labeller"
+	ClusterRoleBindingName   = "kubevirt-node-labeller"
+	nfdVolumeName            = "nfd-source"
+	nfdVolumeMountPath       = "/etc/kubernetes/node-feature-discovery/source.d/"
+	configMapVolumeName      = "cpu-config"
+	configMapVolumeMountPath = "/config"
+)
+
+var commonLabels = map[string]string{
+	"app": "kubevirt-node-labeller",
+}
+
+var cpuPluginConfigmap = `
+    obsoleteCPUs:
```

Comment:
Is this being created properly with leading spaces?

Reply:
Thanks for pointing this out. Made the necessary modifications.